spec/fluent/plugin/out_logtail_spec.rb

```diff
@@ -52,7 +72,23 @@ def format(tag, time, record)
     end
 
     it "handle auth failures" do
-      stub = stub_request(:post, "https://in.logtail.com/").to_return(:status => 403, :body => "", :headers => {})
+      stub = stub_request(:post, "https://in.logs.betterstack.com/").to_return(:status => 403, :body => "", :headers => {})
+
+      driver.emit(record)
+      driver.run
+
+      expect(stub).to have_been_requested.times(1)
+    end
+  end
+
+  describe "#write to cloud" do
```

Comment:
cloud infra is tested `should send a chunked request to the Logtail API`, here's `driver` instead of `cloud_driver`

```suggestion
  describe "#should send a chunked request to the Logtail API using default host" do
```